File: kaleido-gradle-plugin/src/test/fixtures/runtime-smoke/app/src/main/java/example/kaleido/runtime/MainActivity.java
package example.kaleido.runtime;

import android.app.Activity;
import android.os.Bundle;
import android.widget.TextView;

public final class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle state) {
        super.onCreate(state);
        var view = new TextView(this);
        view.setText("Kaleido runtime smoke");
        setContentView(view);
    }
}

File: kaleido-gradle-plugin/src/test/fixtures/runtime-smoke/app/src/main/kotlin/example/kaleido/runtime/MainActivity.kt
package example.kaleido.runtime

import android.app.Activity
import android.os.Bundle
import android.widget.TextView

class MainActivity : Activity() {
    override fun onCreate(state: Bundle?) {
        super.onCreate(state)
        val view = TextView(this)
        view.text = "Kaleido runtime smoke"
        setContentView(view)
    }
}
